db/migrate/20140604130339_create_hubstats_pull_requests.rb

```diff
@@ -1,7 +1,6 @@
 class CreateHubstatsPullRequests < ActiveRecord::Migration
   def change
     create_table :hubstats_pull_requests do |t|
-      t.integer :id
```

Comment:
Yes. Rails 4 automatically makes an `id` integer primary key, and so when we write this into the migration, it thinks we want to overwrite the automatically generated one. We could leave this and add an extra line at the end, but there's no point because we _do_ want the primary key to be `id`.